.github/workflows/tests.yml

```diff
@@ -384,3 +389,39 @@ jobs:
       run: |
         cd ~/spinalcordtoolbox
         ./.ci.sh -t
+
+  windows-native:
+    name: Windows (Native)
+    runs-on: windows-2019
+    defaults:
+      run:
+        shell: cmd
+    steps:
+    - name: Get branch name (merge)
+      if: github.event_name != 'pull_request'
+      shell: bash
+      run: echo "BRANCH_NAME=$(echo ${GITHUB_REF#refs/heads/})" >> $GITHUB_ENV
+    - name: Get branch name (pull request)
+      if: github.event_name == 'pull_request'
+      shell: bash
+      run: echo "BRANCH_NAME=$(echo ${GITHUB_HEAD_REF})" >> $GITHUB_ENV
+    - uses: actions/checkout@v2
+      with:
+        path: checkout/spinalcordtoolbox
+    - name: Install SCT
+      # NB: In real-world usage, it's assumed that the user would download the
+      # install_sct.bat file from the Downloads page, and _not_ the full repo.
+      run: |
+        set HOMEPATH=%GITHUB_WORKSPACE%
+        cd checkout/spinalcordtoolbox
+        .\install_sct.bat %BRANCH_NAME%
+    - name: Add SCT to path
+      # NB: I'm not sure what GHA's syntax is for cmd.exe, so we use bash just for this one change
+      # In a user install, the user would perform this step using the Windows PATH-changing GUI.
```

Comment:
> A possible caveat: it looks like the Github Actions runner runs `cmd` with the `/d` switch, which [disables execution of AutoRun commands](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/cmd). So even if we set `Autoexec.nt`, we probably also need to set `$GITHUB_PATH` as done here.

Caveat 2:

> If you place the path command in your Autoexec.nt file, the Windows operating system automatically appends the specified MS-DOS subsystem search path every time you log on to your computer. _**Cmd.exe does not use the Autoexec.nt file.**_ When started from a shortcut, Cmd.exe inherits the environment variables set in My Computer/Properties/Advanced/Environment. 
> 
> (Emphasize mine)

 😢... 

I feel this, though. I wanted a programmatic way to do this too, like what we currently do with `.bashrc`, etc.

At the same time, from a security perspective, it's nice that this is a step the user has some control over? I just hope it doesn't trip anyone up. 😅

Reply:
I've added what I hope are clear steps for manually setting the PATH in [fd9e06c](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3701/commits/fd9e06c73d2e0b013305621618d04e7d5a6f8ee4).

Reply:
Huh. On closer inspection, it looks like this isn't showing up in GHA's logs? https://github.com/spinalcordtoolbox/spinalcordtoolbox/runs/5383645638?check_suite_focus=true#step:5:436

However, it does in my own Command Prompt. I'm curious to see if helps to swap the steps so that `deactivate` comes _after_. 🤔

Reply:
Yep, [8b400a1](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3701/commits/8b400a11dcaacbc5eb3fa57723f7ea6575e4b324) fixes it:

![image](https://user-images.githubusercontent.com/16181459/156597761-a961fd47-8ce8-44e5-a4ad-2c5e6a324f40.png)

Reply:
Those are great instructions! And if anything is still unclear, the only way to know will be if an actual user gets confused and reports it to us.